.github/workflows/conda-package.yml

```diff
@@ -105,6 +105,7 @@ jobs:
         os: [ubuntu-20.04, ubuntu-latest, windows-latest]
         experimental: [false]
         use_mlir: [false]
+        run_gdb: [true, false]
```

Comment:
Please add exclude for windows with run_gdb true.
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations